pkg/util/util.go

```diff
@@ -1295,6 +1296,31 @@ func PrintThisNoteWhenFinishedApplyingAndStartWaitLoop() {
 	log.Printf("  - You may Ctrl-C at any time to stop the loop and watch it manually.")
 }
 
+// WaitForOperatorDeploymentReady waits for a deployment to be have the desired number of replicas ready
+func WaitForOperatorDeploymentReady(namespace string, name string) {
+	for {
+		deployment := &appsv1.Deployment{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: namespace,
+			},
+		}
+		found := KubeCheckQuiet(deployment)
+		if found {
+			ready := deployment.Status.ReadyReplicas == deployment.Status.Replicas
+			if !ready {
+				log.Printf("⏳ Waiting for %s deployment to be ready", name)
```

Comment:
Will this be printed every second (till it is ready)? 
If yes, we should avoid overloading the logs

Reply:
This is no to the logs. These are messages for the cli install command, same as we always had. I just wait now before creating the system